cmp/example_test.go

```diff
@@ -92,28 +92,28 @@ func ExampleDiff_testing() {
 	for _, test := range tests {
 		AddCrew(test.before, test.name, test.title)
 		if diff := cmp.Diff(test.before, test.after); diff != "" {
```

Comment:
Perhaps:
`s/before/want/g`
`s/after/got/g`
to make it more consistent with the documented string.

Reply:
This example is quite confusing. I tried to make the change you suggested, but it becomes quite weird to mutate a field called `got` to have the result. WDYT?

Reply:
We can leave it as is.